guide/07-working-with-bigdata/10-extending_geoanalytics_with_custom_tools.ipynb

```diff
@@ -0,0 +1,434 @@
+{
```

Comment:
<p><a href="https://developers.arcgis.com/python/sample-notebooks/forecasting-pm2/" target="_blank">This sample notebook</a> can be referred to for further information on the use case above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>This instead of thie</p>